### PR TITLE
updated base for passing, quicker builds on semaphore

### PR DESCRIPTION
**Pull request by danackerson**
@joshuadeguzman here's a cleaner base for Mac builds and [article #5](https://github.com/semaphore-community/joshuadeguzman/pull/1)

### CODE REVIEW

**Review by bogyo210 — APPROVED** [review]

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -1,68 +1,82 @@
...
-        - name: Edit Existing Item Test
+        - name: Edit exist item
           commands:
             - flutter test integration_test/edit_existing_item_test.dart
-        - name: Delete Existing Item Test
+        - name: Delete item
           commands:
             - flutter test integration_test/delete_existing_item_test.dart
+    run:
+      when: 'change_in(''/lib'', {default_branch: ''main'', pipeline_file: ''ignore''}))'
```
Comment (bogyo210) [inline .semaphore/semaphore.yml]: Please remove ) at the end.

File: .semaphore/semaphore.yml [hunk 2]
```diff
@@ -1,68 +1,82 @@
...
+  - name: Build artifact
+    task:
       prologue:
         commands:
           - checkout
           - 'cache restore flutter-packages-$SEMAPHORE_GIT_BRANCH-$(checksum pubspec.yaml),flutter-packages-$(checksum pubspec.yaml),flutter-packages'
-          - sdkmanager "platform-tools" "platforms;android-29" "build-tools;30.0.0" "emulator"
```
Comment (joshuadeguzman) [inline .semaphore/semaphore.yml:59]: Is this passing? The following open an emulator to run the integration tests.

File: lib/features/home/home_viewmodel.dart [hunk 3]
```diff
@@ -1,3 +1,5 @@
+// ignore_for_file: public_member_api_docs
```
Comment (joshuadeguzman) [inline lib/features/home/home_viewmodel.dart:1]: Add this `public_member_api_docs: false` instead to the `analysis_options.yaml` file.

**Review by joshuadeguzman — CHANGES_REQUESTED** [review]
LGTM, small nits.

File: .semaphore/semaphore.yml (hunk 2 above)
Comment (danackerson) [reply .semaphore/semaphore.yml:59]: this line isn't necessary as this semaphore.yml file is for an iOS build

**Review by bogyo210 — CHANGES_REQUESTED** [review]

**Review by bogyo210 — APPROVED** [review]

**Pull request merged**